query/src/storages/fuse/operations/commit.rs

```diff
@@ -221,18 +235,23 @@ impl FuseTable {
         new_snapshot_location: String,
     ) -> Result<UpsertTableOptionReply> {
         let table_id = tbl_id.table_id;
-        let table_version = tbl_id.version;
         let catalog = ctx.get_catalog();
-        catalog
-            .upsert_table_option(UpsertTableOptionReq::new(
-                &TableIdent {
-                    table_id,
-                    version: table_version,
-                },
-                FUSE_OPT_KEY_SNAPSHOT_LOC,
-                new_snapshot_location,
-            ))
-            .await
+
+        let req = UpsertTableOptionReq {
+            table_id,
+            seq: MatchSeq::Exact(tbl_id.version),
+            options: [
+                (
+                    FUSE_OPT_KEY_SNAPSHOT_LOC.to_owned(),
+                    Some(new_snapshot_location),
+                ),
+                (FUSE_OPT_KEY_SNAPSHOT_VER.to_owned(), Some("1".to_owned())),
```

Comment:
I think we should hard code some default value for the current like:
CURRENT_SNAPSHOT_VER = 1
...

Reply:
Right & thanks!!